mapper/orm/src/main/java/org/hibernate/search/mapper/orm/mapping/HibernateOrmMapping.java

```diff
@@ -8,15 +8,38 @@
 
 import javax.persistence.EntityManager;
 
+import org.hibernate.search.mapper.orm.session.HibernateOrmSearchManager;
+import org.hibernate.search.mapper.orm.session.HibernateOrmSearchManagerBuilder;
 import org.hibernate.search.mapper.pojo.mapping.PojoWorkPlan;
-import org.hibernate.search.mapper.pojo.mapping.PojoMapping;
 
-public interface HibernateOrmMapping extends PojoMapping {
+public interface HibernateOrmMapping {
 
 	HibernateOrmSearchManager createSearchManager(EntityManager entityManager);
 
 	HibernateOrmSearchManagerBuilder createSearchManagerWithOptions(EntityManager entityManager);
 
+	/**
+	 * @param type a Java type
```

Comment:
Don't you usually have a capital and a dot?

Reply:
Done